assessments/assessment-3/solution.js

```diff
@@ -0,0 +1,46 @@
+//---------------------------------------------\\
+        // Challenge 1
+
+var challenge1 = "I Love JavaScript";
+var new1 = (challenge1.replaceAll(" ", ''))
+console.log(new1);
+
+
+//---------------------------------------------\\
+        // Challenge 2
+
+var javascript = "javascript";
+var challenge2 = (javascript.replace("javascript", 'JavaScript'))
+console.log(challenge2);
+
+
+//---------------------------------------------\\
+        // Challenge 3
+
+var fullName = "John Doe"
+var initial = "JD"
+console.log(fullName);
+console.log(initial);
+
+
+//---------------------------------------------\\
+        // Challenge 4
+
+var alphabet = "abcdefghijklmnopqrstuvwxyz";
+var alphabetWithoutMyNameFirstCharacter = alphabet.replace("a", "")
```

Comment:
But your name starts with "A". Your code does not work with uppercase. Works only with lowercase